### ISSUE TITLE
Support for event messages.

### ISSUE THREAD

**Issue by mattcollier** [rusty-celery/rusty-celery#221]
Event messages are described here: https://docs.celeryproject.org/en/stable/internals/protocol.html#event-messages

Have I overlooked it, or is it the case that there is currently no support for conveying task return values back to producers by way of the message broker?

If this feature existed, I would think it would be around here:

https://github.com/rusty-celery/rusty-celery/blob/133ec2d9d1955bf7802032f446c9f9e973e55efe/src/app/trace.rs#L94-L95

**Comment by epwalsh** [comment]
> Have I overlooked it, or is it the case that there is currently no support for conveying task return values back to producers by way of the message broker?

This is true. But it's on the roadmap to implement [backends](https://docs.celeryproject.org/en/stable/userguide/tasks.html?highlight=backend#result-backends).
